src/app/redux/TransactionReducer.js

```diff
@@ -0,0 +1,187 @@
+import {fromJS, Map} from 'immutable';
+
+// Action constants
+const CONFIRM_OPERATION = 'transaction/CONFIRM_OPERATION';
+const HIDE_CONFIRM = 'transaction/HIDE_CONFIRM';
+export const BROADCAST_OPERATION = 'transaction/BROADCAST_OPERATION';
+export const UPDATE_AUTHORITIES = 'transaction/UPDATE_AUTHORITIES';
+export const UPDATE_META = 'transaction/UPDATE_META';
+const ERROR = 'transaction/ERROR';
+const DELETE_ERROR = 'transaction/DELETE_ERROR';
+const SET = 'transaction/SET';
+const REMOVE = 'transaction/REMOVE';
+// Saga-related
+export const RECOVER_ACCOUNT = 'transaction/RECOVER_ACCOUNT';
+
+const defaultState = fromJS({
+    operations: [],
+    status: { key: '', error: false, busy: false, },
+    errors: null
+});
+
+export default function reducer(state = defaultState, action) {
+    const payload = action.payload;
+
+    switch (action.type) {
```

Comment:
Should these case statements be constants? Should they be exportable? 
Alphabetize.

Reply:
oops yes they should be constants.. changing this now.